docs/en/02_Developer_Guides/01_Templates/11_Partial_Template_Caching.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Partial Template Caching
+summary: Cache a section of a template Reduce rendering time with cached templates and understand the limitations of the ViewableData object caching.
+icon: tags
+---
+
+## Partial template caching
+
+Partial template caching is a feature that allows caching of rendered portions of templates. Cached content
+is fetched from a [cache backend](../performance/caching), instead of being regenerated repeatedly.
+
+
+### Base syntax
+
+Here is an example of the basic usage. This is not a definitive example of the syntax, but it shows the most
+common use case.
+
+[note]
+See also [Complete Syntax definition](#complete-syntax-defintition) section
+[/note]
+
+```ss
+<% cached $CacheKey if $Condition %>
+  <!-- Content block -->
+<% end_cached %>
+```
+
+The key parts are `$CacheKey`, `$Condition` and `<!-- Content block -->`.  
+The following sections explain every one of them in more detail.
+
+
+#### $CacheKey
+
+The `$CacheKey` is a string identifying the block content within the cache storage.
+
+The form is `[$key[, $...]]`, where `$key` is any valid template expression and you can have any number of them delimited by comma.
```

Comment:
I don't understand this line

Reply:
fair enough, I'll think how to rewrite that.
This is taken from PHP documentation syntax. They define function arguments like that.